contracts/src/v0.8/ccip/test/pools/GHO/GhoTokenPoolRemoteE2E.t.sol

```diff
@@ -0,0 +1,446 @@
+    vm.startPrank(AAVE_DAO);
+    GhoToken(address(srcGhoToken)).grantRole(GhoToken(address(srcGhoToken)).FACILITATOR_MANAGER_ROLE(), AAVE_DAO);
+    GhoToken(address(srcGhoToken)).addFacilitator(address(srcGhoTokenPool), "UpgradeableTokenPool", type(uint128).max);
+    vm.stopPrank();
+    vm.startPrank(OWNER);
+
+    // Add config for source and destination chains
+    UpgradeableTokenPool.ChainUpdate[] memory srcChainUpdates = new UpgradeableTokenPool.ChainUpdate[](1);
+    bytes[] memory remotePoolAddresses = new bytes[](1);
+    remotePoolAddresses[0] = abi.encode(address(dstGhoTokenPool));
+    srcChainUpdates[0] = UpgradeableTokenPool.ChainUpdate({
+      remoteChainSelector: DEST_CHAIN_SELECTOR,
+      remotePoolAddresses: remotePoolAddresses,
+      remoteTokenAddress: abi.encode(address(dstGhoToken)),
+      outboundRateLimiterConfig: _getOutboundRateLimiterConfig(),
+      inboundRateLimiterConfig: _getInboundRateLimiterConfig()
+    });
+    UpgradeableTokenPool.ChainUpdate[] memory dstChainUpdates = new UpgradeableTokenPool.ChainUpdate[](1);
+    remotePoolAddresses[0] = abi.encode(address(srcGhoTokenPool));
+    dstChainUpdates[0] = UpgradeableTokenPool.ChainUpdate({
+      remoteChainSelector: SOURCE_CHAIN_SELECTOR,
+      remotePoolAddresses: remotePoolAddresses,
+      remoteTokenAddress: abi.encode(address(srcGhoToken)),
+      outboundRateLimiterConfig: _getOutboundRateLimiterConfig(),
+      inboundRateLimiterConfig: _getInboundRateLimiterConfig()
+    });
+    vm.stopPrank();
+    vm.startPrank(AAVE_DAO);
+    srcGhoTokenPool.applyChainUpdates(new uint64[](0), srcChainUpdates);
+    dstGhoTokenPool.applyChainUpdates(new uint64[](0), dstChainUpdates);
+    vm.stopPrank();
+    vm.startPrank(OWNER);
+
+    // Update GHO Token price on source PriceRegistry
+    EVM2EVMOnRamp.DynamicConfig memory onRampDynamicConfig = s_onRamp.getDynamicConfig();
+    IPriceRegistry onRampPriceRegistry = IPriceRegistry(onRampDynamicConfig.priceRegistry);
+    onRampPriceRegistry.updatePrices(_getSingleTokenPriceUpdateStruct(address(srcGhoToken), 1e18));
+
+    // Update GHO Token price on destination PriceRegistry
+    EVM2EVMOffRamp.DynamicConfig memory offRampDynamicConfig = s_offRamp.getDynamicConfig();
+    IPriceRegistry offRampPriceRegistry = IPriceRegistry(offRampDynamicConfig.priceRegistry);
+    offRampPriceRegistry.updatePrices(_getSingleTokenPriceUpdateStruct(address(dstGhoToken), 1e18));
+
+    s_tokenAdminRegistry.proposeAdministrator(address(srcGhoToken), AAVE_DAO);
+    s_tokenAdminRegistry.proposeAdministrator(address(dstGhoToken), AAVE_DAO);
+    vm.stopPrank();
+    vm.startPrank(AAVE_DAO);
+    s_tokenAdminRegistry.acceptAdminRole(address(srcGhoToken));
+    s_tokenAdminRegistry.setPool(address(srcGhoToken), address(srcGhoTokenPool));
+    s_tokenAdminRegistry.acceptAdminRole(address(dstGhoToken));
+    s_tokenAdminRegistry.setPool(address(dstGhoToken), address(dstGhoTokenPool));
+    vm.stopPrank();
+    vm.startPrank(OWNER);
```

Comment:
This makes sense to me for the test suite if we want to prank owner after the setup for the remainder of the tests, but I see below we are pranking owner again, so I think we only need to have this in one place. Maybe more readable just to prank in each individual test as needed. Wdyt?

Reply:
it's to keep the standard of tests consistent across, the entire GHO test suite has owner being the default sender for all tests, i believe we should stick to that